src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockFetcher.scala

```diff
@@ -113,9 +114,10 @@ class BlockFetcher(
       }
 
       fetchBlocks(newState)
+
     case InvalidateBlocksFrom(blockNr, reason, withBlacklist) =>
       val (blockProvider, newState) = state.invalidateBlocksFrom(blockNr, withBlacklist)
-      log.debug("Invalidate blocks from {}", blockNr)
+      log.info("Invalidate blocks from {}", blockNr)
```

Comment:
I still think that these should not be changed to info. a better approach (imo) would be to setup the testnet with the logbacks BlockFetcher entry set to debug. this way we avoid burdening every other usage of mantis with additional logging.

Reply:
That is indeed a valid approach. I do find weird that BlockFetcher has so many debug log entries, while the rest of the application also uses info. Note: I did not change all logs in BlockFetcher to be info

Reply:
I agree that not all of them should be changed to `info`...I'd change the ones indicating failures to `warn` and only important ones to info.

Reply:
I have rolled back the modification in the logs, expect one. I think this can be reviewed better by someone that actually touches the code